src/main/java/UI.java

```diff
@@ -0,0 +1,37 @@
+public class UI {
```

Comment:
I like how you abstracted methods such as printing to make the overall code in the main function less clutered.